ParisBNF/abb/BNFabb233.xml

```diff
@@ -0,0 +1,176 @@
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                        <idno>BnF Éthiopien d'Abbadie 233</idno>
+                  </msIdentifier>
+
+                    <msContents>
+                      <msItem xml:id="ms_i1">
+                          <locus from="1r" to="4v"/>
+                          <title type="incomplete" ref="LIT5982CommOrit" xml:lang="gez"/>
+                          
+                          <msItem xml:id="ms_i1.1">
+                              <locus from="1r" to="4v"/>
+                              <title type="incomplete" ref="LIT5982CommOrit#Genesis" xml:lang="gez">የኦሪት፡ የፍጥረት፡ ትርጓሜ፡</title>
+                          </msItem>
+                        </msItem>
+                    </msContents>
+
+                    <physDesc>
+                        <objectDesc form="Leaf">
+                            <supportDesc>
+                                <support>
+                           <material key="paper"/>
+                        </support>
+                                <extent>
+                                    <measure unit="leaf">4</measure>
+                                    <dimensions type="outer" unit="mm">
+                                      <height>240</height>
+                                      <width>170</width>
+                                    </dimensions>
+                                </extent>
+                                <condition key="other">Four leaves of Oriental paper, no binding.</condition>
+                                </supportDesc>
+
+                             <layoutDesc>
+                                <layout columns="1" writtenLines="19">
+                                <locus from="2r"/>
+                                </layout>
+                             </layoutDesc>
+                        </objectDesc>
+
+                        <handDesc>
+                           <handNote xml:id="h1" script="Ethiopic">
+                              <seg type="script">19th-century script.</seg>
+                              <seg type="ink">Black.</seg>
+                               <desc>Crude; according to Antoine d'Abbadie, written by a <persName ref="ETH1274Betaes">falāšā</persName> scribe.</desc>
```

Comment:
As far as I know, the term 'Falasha' is outdated. Being a quotation by Abbadie, it should have quotation marks.

Reply:
OK, done.